deployments/optimism-goerli/usdc/deploy.ts

```diff
@@ -0,0 +1,129 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { DeploySpec, deployComet, exp, wait } from '../../../src/deploy';
+
+const clone = {
+  op: '0x4200000000000000000000000000000000000042'
+};
+
+const HOUR = 60 * 60;
+const DAY = 24 * HOUR;
+
+const GOERLI_TIMELOCK = '0x8Fa336EB4bF58Cfc508dEA1B0aeC7336f55B1399';
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const deployed = await deployContracts(deploymentManager, deploySpec);
+  await mintTokens(deploymentManager);
+  return deployed;
+}
+
+async function deployContracts(
+  deploymentManager: DeploymentManager,
+  deploySpec: DeploySpec
+): Promise<Deployed> {
+  const trace = deploymentManager.tracer();
+  const ethers = deploymentManager.hre.ethers;
+
+  // Pull in existing assets
+  const USDC = await deploymentManager.existing(
+    'USDC',
+    '0x7E07E15D2a87A24492740D16f5bdF58c16db0c4E',
+    'optimism-goerli'
+  );
+  const WETH = await deploymentManager.existing(
+    'WETH',
+    '0x4200000000000000000000000000000000000006',
+    'optimism-goerli'
+  );
+  const WBTC = await deploymentManager.existing(
+    'WBTC',
+    '0xe0a592353e81a94Db6E3226fD4A99F881751776a',
+    'optimism-goerli',
+    'contracts/ERC20.sol:ERC20'
```

Comment:
I added an extra optional param to `dm.existing` to pass in an artifact for contracts that aren't verified on Etherscan.